config/environments/staging.rb

```diff
@@ -8,4 +8,7 @@
 
   # To allow HTTPS on CloudFlare but HTTP on Heroku
   config.action_controller.forgery_protection_origin_check = false
+
+  # Override mailer host
+  config.action_mailer.default_url_options = { host: 'admin-staging.wemeditate.com' }
```

Comment:
Probably makes more sense to just use ENV vars for the `default_url_options`. Could be dealt with separately though.